erizo_controller/erizoClient/src/Stream.js

```diff
@@ -280,25 +280,41 @@ Erizo.Stream = function (spec) {
         }
     };
 
+    that.muteAudio = function (audioActive, callback) {
```

Comment:
having muteAudio with audioActive seems like a bit complex to understand for the purpose of this feature IMO. Consider other options like:

that.muteAudio = function(audioMuted,....

or

that.setAudioActive = function(audioActive,....

or 

that.muteAudio = function(...
that.unmuteAudio = function(...